lerobot-web-fe/src/components/Tooltip.module.css

```
@@ -0,0 +1,76 @@
+.content {
+  padding: 10px 15px;
+  font-size: 15px;
+  line-height: 1;
+  background-color: #f0f0f091;
+  border: 1px solid #fff;
+  box-shadow: 0px 0px 2px 0px #fff;
+  border-radius: 8px;
+  box-shadow:
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

box-shadow is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)